js/git-read-issues/src/githubClient.ts

```diff
@@ -62,12 +67,7 @@ export class GithubClient {
 
                     issuesArr.push(entry);
 
-                    await this.octokit.rest.issues.addLabels({
-                        owner: this.owner,
-                        repo: this.repo,
-                        issue_number: e.number,
-                        labels: [readLabel]
-                    });
+                    await fetch(`${this.URL}/issues/${e.number}/labels`, fetchOptions("POST",this.apiKey,JSON.stringify({"labels": [readLabel]})))
```

Comment:
The `this.URL` is not set causing an invalid URL error, is `setRepo` method not called?

Reply:
fixed with [eae1454](https://github.com/scramjetorg/reference-apps/pull/37/commits/eae14543b0201ed8c786e7d3ffa767f4f3f5ba66)